protocol-select/README.md

```diff
@@ -69,31 +69,11 @@ Select]_ protocol.
 
 ### Improvements over _[Multistream Select]_
 
-- **Downgrade attacks** and **censorship resistance**
-
-  Given that **[Multistream Select]** negotiates a connection's security
-  protocol unencrypted and unauthenticated it is prone to [downgrade attack]s.
-  In addition, a man-in-the-middle can detect that a given connection is used
-  to carry libp2p traffic, allowing attackers to censor such connections.
-
-  **Protocol Select** is combined with a change to the Multiaddr format,
-  advertising the secure channel protocol through the latter instead of
-  negotiating them in-band. Thus [Downgrade attack]s are no longer possible at
-  the protocol negotiation level and a man-in-the-middle can no longer detect
-  a connection being used for libp2p traffic through the negotation process.
-
-- **Connection establishment**
-
-  In addition to making us vulnerable to downgrade attacks, negotiating the
-  security protocol takes one round-trip in the common case with **[Multistream
-  Select]**. On top of that negotiating a stream multiplexer (on TCP) takes
-  another round-trip.
-
-  **Protocol Select** on the other hand depends on security protocols being
-  advertised, thereby eliminating the need for negotiating them. For optimized
-  implementations, stream muxer negotiation will take zero round-trips for the
-  client (depending on the details of the cryptographic handshake protocol). In
-  that case, the client will be able to immediately open a stream after
+- **Protocol Select** requires security protocols to be advertised, and doesn't 
```

Comment:
```suggestion
- **Connection establishment**
  
  **Protocol Select** requires security protocols to be advertised, and doesn't 
```

I would like to be consistent with the other bullet points, namely that each of them has a heading. Any objections to simply keep the previous one?